packages/sharing-dialog/src/Sharing.component.js

```diff
@@ -2,7 +2,7 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Divider from '@material-ui/core/Divider';
 import Typography from '@material-ui/core/Typography';
-import { Heading } from '@dhis2/d2-ui-core';
+import Heading from '@dhis2/d2-ui-core/build/es/headings/Heading.component';
```

Comment:
After flatpacking in #327 this will be: 

```
import Heading from '@dhis2/d2-ui-core/es/headings/Heading.component';
```

If we drop CJS we can make it:

```
import Heading from '@dhis2/d2-ui-core/headings/Heading.component';
```

Reply:
But we can merge this first and I'll fix my PR.